### PR TITLE
Rh user profile

### PR DESCRIPTION
**Pull request by reherr**

### CODE REVIEW

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
   </div>
 
   <div>
-    <%= form.submit %>
+    <%= form.submit class: "btn btn-outline-secondary btn-block" %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:21]: Great job refactoring your code :)

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:21]: The image tag here has empty src and alt attributes. Its good practice to add alt description for accessibility reasons and maybe a default image src if the user did not create one. For this assignment it was not required so this is more informational for future purposes :)

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                </div>
+                <div class="flex-grow-1 ms-3">
+                  <h5 class="mt-0"><%= comment.author.username %></h5>
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:24]: great use of associations!

File: config/routes.rb [hunk 4]
```diff
@@ -8,8 +8,7 @@
   resources :follow_requests
   resources :likes
   resources :photos
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:12]: Nice custom route!

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by!(username: params.fetch(:username))
```
Comment (JoseMDO) [inline app/controllers/users_controller.rb:3]: great job!

**Review by JoseMDO — APPROVED** [review]
Great job Rebecca! Pr looks great!

File: app/controllers/comments_controller.rb
Comment (borvux) [inline app/controllers/comments_controller.rb:1]: good job on noticing the change

File: app/controllers/users_controller.rb
Comment (borvux) [inline app/controllers/users_controller.rb:1]: nice job on using the find_by! to help with error handling

File: app/views/comments/_form.html.erb (hunk 1 above)
Comment (borvux) [reply app/views/comments/_form.html.erb:21]: I agree !

File: app/views/users/show.html.erb (hunk 2 above)
Comment (borvux) [reply app/views/users/show.html.erb:21]: good to know

File: config/routes.rb (hunk 4 above)
Comment (borvux) [reply config/routes.rb:12]: yes, good job!

**Review by borvux — APPROVED** [review]

**Pull request merged**